### ISSUE TITLE
UserWarning: None of the inputs have requires_grad=True. Gradients will be None   warnings.warn("None of the inputs have requires_grad=True. Gradients will be None")

### ISSUE THREAD

**Issue by TkonstAL** [adieyal/sd-dynamic-prompts#227]
![uip08](https://user-images.githubusercontent.com/91704308/215615398-f124722f-a00c-4350-b04d-c0c988aad0c5.png)
Already up to date.
venv "D:\SD\stable-diffusion-webui\venv\Scripts\Python.exe"
Python 3.10.8 (tags/v3.10.8:aaaf517, Oct 11 2022, 16:50:30) [MSC v.1933 64 bit (AMD64)]
Commit hash: 2c1bb46c7ad5b4536f6587d327a03f0ff7811c5d
Installing requirements for Web UI
Installing requirements for scikit_learn

Installing requirements for Prompt Gallery

Installing sd-dynamic-prompts requirements.txt

Launching Web UI with arguments: --xformers --no-half --autolaunch --no-half-vae
Loading booru2prompt settings
[AddNet] Updating model hashes...
0it [00:00, ?it/s]
Hypernetwork-MonkeyPatch-Extension found!
Loading weights [70346f7a1e] from D:\SD\stable-diffusion-webui\models\Stable-diffusion\artErosAerosATribute_aerosNovae.safetensors
Creating model from config: D:\SD\stable-diffusion-webui\configs\v1-inference.yaml
LatentDiffusion: Running in eps-prediction mode
DiffusionWrapper has 859.52 M params.
Applying xformers cross attention optimization.
Textual inversion embeddings loaded(136): 2000ccplus, ACKass1, akii_marie_rose, ALATSuckFace, Alettaocean, ambphillipsx, AnaD, AriGra3V-383, bfold3, BiL13L1SH, bimbostyle, blowjobpose, brittney-williams-art, bs_swaisy, bukkakAI, cessa-8100, christymack, corneo_anal, corneo_black_cat_dva, corneo_bound_missionary, corneo_brigitte, corneo_covering_breasts_arms_crossed, corneo_covering_breasts_one_arm, corneo_covering_breasts_two_hands, corneo_cowgirl, corneo_makima, corneo_nipple_rings, corneo_paizuri, corneo_pov_anal, corneo_pov_oral, corneo_runny_makeup, corneo_side_deepthroat, corneo_side_doggy, corneo_spitroast, corneo_tentacle_sex, corneo_tifa_classic, corneo_widowmaker, corneo_x_pasties, corneo_yor_forger, devilhs-69100, devmgf-8500, dpthrt, emb-babs, EmWat69, female-kpop-singer, fumio-8000 (1), fumio-8000, GITSEllie, GITSKrystenR, hana_uzaki, hotfantasyface, HyperShading, HyperStylizeLiteA, HyperStylizeLiteB, HyperStylizeLiteC, HyperStylizeLiteD, HyperStylizeLiteE, HyperStylizeLiteF, HyperStylizeV6, HyperStylizeVA, HyperStylizeVB, HyperStylizeVC, HyperStylizeVD, HyperStylizeVE, HyperStylizeVF, iranon-64200, janrock-93000, JeBeFlash2, joemad, jujunaught-66600, LOP, LPB2k1, lucy_kushinadaV2, MakeItPopVA, MakeItPopVB, MakeItPopVC, MakeItPopVD, MakeItPopVE, MakeItPopVF, marin_7, marisa-4500, meganfox, microkini, micromini, missymuscle-8100, mulpty2, musctonk-8100, nami, ndc-8000, neutral_selen_tatsuki, neutral_shylily, NewLaraCroftV2, not soojin, NovaFrogStyle, ohmyfrick_yogapuss, optional_styleAI, paradoxical-back-view-bj, pomu_rainpuff, povblowjob3v3, powergirl-8100, PureErosFace_V1, realcumAI, reiq-37400, RHillcrest, rld, robin_v2_1600, RSC, russian, saoirseronan, scrltff7v1, shantae_styleAI, shinybimbo, shirtp5, speedl00ver-8000, Style-Autumn, Style-Empire, Style-Glass, Style-Glorious, Style-Hamunaptra, Style-Italy, Style-Japan, Style-Kilt, Style-Moana, style-nagel, Style-Petal, Style-Psycho, Style-Renaissance, Style-Winter, thegoldensmurf-8100, tifa_4900, vanillaware-32900, very2, wano_style_30100, yamato_5000, yor_briar, _SamDoesArt2_
Textual inversion embeddings skipped(16): CinemaHelper, flonixsdviewv3, FloralMarble, InkPunk768, Inksplat768, IpScifiHelper, jah_sidelight, no_unrealistic768, painted_abstract, PaintStyle3, PaintStyle4, PhotoHelper, PortraitHelper, rzviceb6, SCG768-Illustrate, SDA768
Model loaded in 41.9s (load weights from disk: 0.1s, create model: 0.9s, apply weights to model: 34.0s, move model to device: 3.4s, load textual inversion embeddings: 3.4s).
Textual inversion embeddings loaded(0):
D:\SD\stable-diffusion-webui\venv\lib\site-packages\gradio\components.py:2263: UserWarning: The `bytes` type is deprecated and may not work as expected. Please use `binary` instead.
  warnings.warn(
INFO:     Started server process [35544]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
ERROR:    [Errno 10048] error while attempting to bind on address ('127.0.0.1', 5173): обычно разрешается только одно использование адреса сокета (протокол/сетевой адрес/порт)
INFO:     Waiting for application shutdown.
INFO:     Application shutdown complete.
D:\SD\stable-diffusion-webui\venv\lib\site-packages\gradio\deprecation.py:43: UserWarning: You have unused kwarg parameters in Row, please remove them: {'equal_height': True}
  warnings.warn(
D:\SD\stable-diffusion-webui\venv\lib\site-packages\gradio\deprecation.py:43: UserWarning: You have unused kwarg parameters in Button, please remove them: {'full_width': True}
  warnings.warn(
Running on local URL:  http://127.0.0.1:7861

To create a public link, set `share=True` in `launch()`.
  0%|                                                                                           | 0/20 [00:00<?, ?it/s]D:\SD\stable-diffusion-webui\venv\lib\site-packages\torch\utils\checkpoint.py:31: UserWarning: None of the inputs have requires_grad=True. Gradients will be None
  warnings.warn("None of the inputs have requires_grad=True. Gradients will be None")

**Comment by mezotaken** [comment]
That's not the error author wanted to report.
It crashes later at that line 
https://github.com/adieyal/sd-dynamic-prompts/blob/49eaa7997e80ae479e17e63752110e7ec72b1204/sd_dynamic_prompts/dynamic_prompting.py#L353

with error `Script has no attribute _pnginfo_saver`
So i assume something's been broken with last a1111 updates here
https://github.com/adieyal/sd-dynamic-prompts/blob/49eaa7997e80ae479e17e63752110e7ec72b1204/sd_dynamic_prompts/dynamic_prompting.py#L110-L118

Author will probably post adequate logs later. (Rerouted from here https://github.com/AUTOMATIC1111/stable-diffusion-webui/issues/7406 after fix of hypernets-monkeypatch and encounter of this one)

**Comment by adieyal** [comment]
Could you please try the latest version and let me know if it fixes the issue.

**Comment by mezotaken** [comment]
I tried both previous version and this one on clean a1111 install and was not able to reproduce the error. Probably had to do with author having 20 extensions installed or some unique settings, or the way of installation. I just witnessed the error on the live stream, it did happen somehow.

**Comment by adieyal** [comment]
thanks for letting me know - I'll close this issue but re-open it if it happens again.
